feature-utils/poly-look/test/react-components/contexts/poly-import.test.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { fireEvent, render, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { PolyImportProvider } from "../../../src/react-components";
+import {
+  MockParentContext,
+  MockParentContextProvider,
+} from "./mocks/parent-context-mock";
+import MockDataAccount from "./mocks/data-account-mock";
+import MockPolyImportComponent from "./mocks/poly-import-component-mock";
+import { mockDataImporters } from "./mocks/data-importers-mock";
+import { mockFiles } from "./mocks/pod-mock";
+/**
+ * @jest-environment jsdom
+ */
+const mockComponent = (
```

Comment:
Right, I would say that most of these mocks are technically fixtures. Mocks are used to substitute for "real" objects, but what you're doing here is creating a structure that you're going to test, thus a fixture, actualy. Not a big deal, anyway.